scw/errors.go

```diff
@@ -185,34 +195,62 @@ func (e *InvalidArgumentsError) GetRawBody() json.RawMessage {
 	return e.RawBody
 }
 
-// InvalidRequestError are only returned by the compute API
+// InvalidRequestError is only returned by the instance API.
 // Warning: this is not a standard error.
 type InvalidRequestError struct {
+	Message string `json:"message"`
```

Comment:
I don't remember seing this message field in the response, but leave it there just in case it happens. 👍

Reply:
In this kind of errors: https://github.com/scaleway/scaleway-sdk-go/pull/278/files#diff-b67d56a36244dc1228b3d114d7fdb93fR77